docs/design/features/crossgen2-compilation-structure-enhancements.md

```diff
@@ -37,27 +39,29 @@ The default value of the CompilationVersionBubble flag would be dependent on how
 
 ## Specifying the version bubble on the crossgen2 command line
 
-There are 3 sets of files to pass to crossgen2.
+There are 3 sets of files to pass to crossgen2:
 
-1. The set of files that are referenceable by the compilation. These are specified via the --reference switch.
-2. The set of files that comprise the set of assemblies being compiled at once. These are specified via the --input-file-path switch. (The first assembly specified in this list describes module actually to be compiled and output. When we add support for composite R2R images, we will produce a single module file from all of these input assemblies at once.)
-3. The set of modules that comprise the version bubble tied to the assembly being compiled. The specification of this is complicated and unclear at this time. My current idea is to have two modes. Input bubble mode (where the set of input files matches the version bubble), and all mode (where all assemblies are considered to be part of the bubble. For framework ahead of time compilation scenarios such as containers and such where we may be compiling the framework and supporting scenarios where not all files in the framework are necessarily unified in the bubble, we may need an exclude function to exclude specific binaries.
+- The set of files that are referenceable by the compilation. These are specified via the --reference switch.
```

Comment:
Why was the numbered list problematic?

Reply:
It says that there are three things and then lists the three things; numbers make sense there... it's not about order but about counting.

Reply:
Reverted those.